lib/git.sh

```
@@ -133,6 +255,22 @@ require_allowed_signer() {
   local rows=""
+  local header_cols=(Commit Status Service Env Author Date)
+  local header_line
+  local gum_bin="${GUM:-gum}"
+  local gum_available=0
+  local gum_missing_message='shiplog: %s not found; falling back to plain output\n'
+
+  if command -v "$gum_bin" >/dev/null 2>&1; then
+    gum_available=1
+  fi
+  if [ "$gum_available" -ne 1 ]; then
+    printf "$gum_missing_message" "$gum_bin" >&2
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)